Client/Ecosia/UI/NTP/Customization/NTPCustomizationCell.swift

```diff
@@ -0,0 +1,71 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+final class NTPCustomizationCell: UICollectionViewCell, NotificationThemeable, ReusableCell {
+    struct UX {
+        static let buttonHeight: CGFloat = 40
+        static let horizontalInset: CGFloat = 16
+        static let verticalInset: CGFloat = 12
+        static let imageTitlePadding: CGFloat = 4
+    }
+    
+    weak var delegate: NTPCustomizationCellDelegate?
+    
+    private lazy var button: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setTitle(.localized(.customizeHomepage), for: .normal)
+        button.titleLabel?.font = .preferredFont(forTextStyle: .callout)
```

Comment:
wondering if you need to set `adjustsFontSizeToFitWidth` on the `titleLabel` to make it scale? I'd think so

Reply:
I guess it is indeed relevant for smaller screens, will add it 👍